src/discovery/azure/Akka.Discovery.Azure/Actors/HeartbeatActor.cs

```diff
@@ -0,0 +1,121 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="HeartbeatActor.cs" company="Akka.NET Project">
+//      Copyright (C) 2013-2022 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Akka.Actor;
+using Akka.Event;
+using Akka.Util.Internal;
+
+namespace Akka.Discovery.Azure.Actors
+{
+    internal sealed class HeartbeatActor: UntypedActor, IWithTimers
+    {
+        public static Props Props(AzureDiscoverySettings settings, ClusterMemberTableClient client)
```

Comment:
Good - need the `Deploy.Local` so there's no problems when `serialize-creators` is turned on. You've done this consistently throughout this library. Nice work.